CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
I think we should either completely adhere to the format including all sectoins or should mention which sections we use.

Reply:
I've added all the missing sections.
https://github.com/orca-services/cakephp-feature-flags/pull/20/commits/4a2a8eecc627ac4aff5434f2e05a78258e7fd60a